paper-checkbox.html

```diff
@@ -186,9 +185,6 @@
       }
 
       /* disabled state */
-      :host([disabled]) {
-        pointer-events: none;
-      }
```

Comment:
(Same as [here](https://github.com/PolymerElements/paper-radio-button/pull/87#discussion_r53222739), [IronControlState already handles this.](https://github.com/PolymerElements/iron-behaviors/blob/9ffe111/iron-control-state.html#L87))